cpp/comms/std/src/ucp_helper.h

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <ucp/api/ucp.h>
```

Comment:
@Akshay-Venkatesh, it would be great if we could get your blessing on the contents of this file, as well as `cuML_std_comms_impl.cpp` file before this gets merged (hopefully before this coming weekend).

Reply:
Will do that.

Reply:
Thank you for reviewing this, Akshay. Your feedback is very useful. I have implemented the changes, with the exception of the tag change, which I need to understand better.